site/index.ejs

```diff
@@ -1,90 +1,113 @@
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-60x60.png" type="image/png" sizes="60x60">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-72x72.png" type="image/png" sizes="72x72">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-76x76.png" type="image/png" sizes="76x76">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-114x114.png" type="image/png" sizes="114x114">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-120x120.png" type="image/png" sizes="120x120">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-144x144.png" type="image/png" sizes="144x144">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-152x152.png" type="image/png" sizes="152x152">
+  <link rel="apple-touch-icon" href="/assets-honestly/favicons/apple-touch-icon-180x180.png" type="image/png" sizes="180x180">
+  <link rel="icon" href="/assets-honestly/favicons/favicon-96x96.png" type="image/png" sizes="96x96">
+  <link rel="icon" href="/assets-honestly/favicons/favicon-192x192.png" type="image/png" sizes="192x192">
+  <link rel="icon" href="/assets-honestly/favicons/favicon-16x16.png" type="image/png" sizes="16x16">
+  <link rel="icon" href="/assets-honestly/favicons/favicon-32x32.png" type="image/png" sizes="32x32">
+  <link rel="icon" href="/assets-honestly/favicons/favicon-194x194.png" type="image/png" sizes="194x194">
+  <link rel="manifest" href="/manifest.json">
+  <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7838134/6842972/css/fonts.css" />
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/UAParser.js/0.7.17/ua-parser.min.js" charset="utf-8"></script>
+  <script type="text/javascript">
+    var uaSniffer = new UAParser();
+    var browser = uaSniffer.getBrowser();
+    if (window.location.pathname !== '/<%= process.env.URL_BASENAME || '' %>browser-not-supported/' ) {
+      if (browser.name === 'IE' && browser.major < 10) {
+        window.location.pathname = '/<%= process.env.URL_BASENAME || '' %>browser-not-supported/';
+      }
+    }
+  </script>
+  <title>
+    <% if (typeof title !== 'undefined') {%>
+      <%= title %>
+        <% } %>
+  </title>
+  <% if (typeof meta !== 'undefined') { %>
+    <%= meta %>
+      <% } %>
+        <% if (typeof cssPath !== 'undefined') { %>
+          <link rel="stylesheet" href="<%= cssPath %>" type="text/css" charset="utf-8">
+          <% } %>
+            <% if (typeof tracking !== 'undefined' && tracking) { %>
+              <script async type="text/javascript" src="https://secure.leadforensics.com/js/77932.js"></script>
+              <noscript>
+                <img src="https://secure.leadforensics.com/77932.png" style="display: none;" />
+              </noscript>
+              <% } %>
+                <script>
```

Comment:
This formatting is actually wrong (for title and the script after it) ... don't think prettier handles ejs files well. Maybe we shouldn't be running prettier on this file, can we ignore it in config?

Reply:
added .prettierignore and excluded ejs files